packages/block-editor/src/components/block-types-list/index.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import { Composite, useCompositeState } from 'reakit';
```

Comment:
It adds nearly 10kB to the module.

Reply:
yes, I was wondering about that. the solution could be to make it part of the `@wordpress/components` API but then it becomes something we need to ensure BC for. Always a struggle to decide what's best in these situations.

Reply:
Let's improve the experience first and see how it might get improved later then.

Reply:
I'm hoping we can balance these kbits loss by refactoring some of our components to rely on reakit like NavigableToolbar/NavigableContainer...

Reply:
Yes, it's essentially what `Composite` does 👍